tests/test_fields.py

```diff
@@ -1153,6 +1160,41 @@ class TestDateTimeField(FieldValues):
     field = serializers.DateTimeField(default_timezone=timezone.UTC())
 
 
+class TestIso8601StrictDateTimeField(FieldValues):
+    valid_inputs = {
+        '2001-01-01 13:00': datetime.datetime(2001, 1, 1, 13, 00, tzinfo=timezone.UTC()),
+        '2001-01-01 13:00:00.123456': datetime.datetime(2001, 1, 1, 13, 00, tzinfo=timezone.UTC()),
```

Comment:
Shouldn't the inputs with fractional seconds fail? How does this behavior differ from non-strict?